### ISSUE TITLE
Question about fine-tuning

### ISSUE THREAD

**Issue by l-salewski** [magic-research/PLLaVA#23]
Dear Authors,

thank you for making this code public! I am trying to fine-tune the final PLLaVA model, but when loading the weights the predictions are completely broken. I followed [this comment to load the pllava 7b weights](https://github.com/magic-research/PLLaVA/issues/9#issuecomment-2084321560).

When I use `processor.batch_decode` to decode the first predictions in the training loop [before any gradient update has been made](https://github.com/magic-research/PLLaVA/blob/e2032aa7c7d8c860520d4e357e32040e630da82c/tasks/train/train_pllava_nframe_accel.py#L446), the predictions are complete garbage, e.g. `ierp2 P What:,.ly.</s>ellow</s>.-.. ce.2...0 lightsed0a` and so on. On the other hand, running `scripts/demo.sh` with the same input works just fine.

I wonder whether it is related to PEFT nature of the PLLaVA model? I already changed `model.lora_alpha=4` and `model.lora_dropout=0.0
` to match the settings in `scripts/demo.sh`, but that doesn't help. According to the [PEFT documentation](https://huggingface.co/docs/peft/main/en/conceptual_guides/lora) adapters are loaded with load_adapter, instead of loading the state dict, but this fails because there is no `adapter_model.safetensors`.

Thus, I am wondering what is the correct way to load the PLLaVA model for further fine-tuning?

**Comment by gaowei724** [comment]
Hi， have you solved this?
